frontend/test/src/reducer/user.spec.js

```diff
@@ -0,0 +1,55 @@
+import { expect } from 'chai'
+import userReducer from '../../../src/reducer/user.js'
+import {
+  UPDATE,
+  USER_USERNAME,
+  USER_PUBLIC_NAME
+} from '../../../src/action-creator.sync'
+import { getBrowserLang } from '../../../src/helper.js'
+import { PROFILE } from 'tracim_frontend_lib'
+
+const defaultUser = {
+  user_id: -1,
+  logged: null,
+  auth_type: '',
+  timezone: '',
+  profile: PROFILE.user,
+  email: '',
+  is_active: true,
+  avatar_url: null,
+  created: '',
+  public_name: '',
+  lang: getBrowserLang(),
```

Comment:
I don't think you should be using getBrowerLang().
The unit tests aren't run inside a browser, I don't know what to expect from getBrowerLang() but I'm assuming it is undefined or a default value.